src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -318,10 +318,53 @@ int halide_hexagon_remote_get_symbol_v4(handle_t module_ptr, const char* name, i
     return *sym_ptr != 0 ? 0 : -1;
 }
 
+// Thread priority for QURT threads
+// Negative: use the current default (don't explicitly reset it)
+// Positive: the priority needs to be set once the shared runtime is loaded
+int saved_thread_priority = -1;
+
+int halide_hexagon_remote_set_thread_priority(int priority) {
+    // Just save requested priority for now.  The priority can't actually
+    // be set in qurt_thread_pool until the shared runtime has been loaded.
+    saved_thread_priority = priority;
+    return 0;
+}
+
+int halide_hexagon_runtime_set_thread_priority(int priority) {
+    if (priority < 0) {
+        return 0;
+    }
+
+    // Find the halide_set_default_thread_priority function in the shared runtime,
+    // which we loaded with RTLD_GLOBAL.
+    void (*set_priority)(int) = NULL;
+    if (use_dlopenbuf()) {
+        set_priority = (void (*)(int)) halide_get_symbol("halide_set_default_thread_priority");
+    } else {
+        set_priority = (void (*)(int)) mmap_dlsym_libs("halide_set_default_thread_priority");
```

Comment:
It seems like this should always use halide_get_symbol, and halide_get_symbol is the thing that should be calling mmap_dlsym_libs.